src/Features/VisualBasic/Portable/ExtractMethod/VisualBasicSelectionResult.vb

```diff
@@ -74,6 +74,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ExtractMethod
                 selectionChanged)
         End Sub
 
+        Public Overrides Function IsExtractMethodOnSingleStatement() As Boolean
```

Comment:
these are moves.

Reply:
i also think this logic is identical to the C# versions.  i will try to unify them in a followup pr.